tests/_testerlib.livecodescript

```diff
@@ -130,6 +147,39 @@ function TesterParseTestCommandNames pFilename
    return tCommandNames
 end TesterParseTestCommandNames
 
+-- Get a number-indexed array containing the names of all "test"
+-- blocks in pFilename.
+function TesterParseCompilerTestNames pFilename
+   local tScript
+
+   -- Get the contents of the file
+   open file pFilename for "UTF-8" text read
+   if the result is not empty then
+      throw the result
+   end if
+
+   read from file pFilename until end
+   put it into tScript
+
+   close file pFilename
+
+   -- Scan the file for "on Test*" definitions
+   local tTestNames, tCount, tLine, tName
+
+   repeat for each line tLine in tScript
+      if word 1 of tLine is not "%TEST" then
+         next repeat
+      end if
+
+      put word 2 of tLine into tName
+
+      add 1 to tCount
+      put tName into tTestNames[tCount]
+   end repeat
+
+   return tTestNames
+end TesterParseCompilerTestNames
```

Comment:
This seems quite compiler-test-specific, so why not keep it in the compiler test runner for now?  The `_testerlib.livecodescript` library was added to contain code that's shared between the IDE & engine test runner implementations, but this function isn't likely to be useful elsewhere.